### PR TITLE
Added Civilian Naval Units

### PR DESCRIPTION
**Pull request by MustaphaTR**
Adds Tug Boat, Cruise Ship and Coast Guard Boat.
civilian vessels

### CODE REVIEW

File: mod.yaml [hunk 1]
```diff
@@ -74,6 +74,7 @@ Rules:
 	./mods/ra2/rules/aircraft.yaml
 	./mods/ra2/rules/civilian-props.yaml
 	./mods/ra2/rules/trees.yaml
+	./mods/ra2/rules/civilian-naval.yaml
```
Comment (Mailaender) [inline mod.yaml]: Note to myself: I will need to update here when rebasing https://github.com/OpenRA/ra2/pull/28.

**Comment by Phrohdoh** [comment]
```ini
;Coast Guard Boat
[CDEST]
UIName=Name:CDEST
Name=Coast Guard Boat
Primary=155mm
Secondary=ASWLauncher
NavalTargeting=1
Spawns=ASW
SpawnsNumber=1
SpawnRegenRate=400
SpawnReloadRate=150
NoSpawnAlt=yes ; alternate voxel for out of spawns: xxxxWO (CDESTWO)
FireAngle=32
ToProtect=yes
Category=Support
Strength=600
Naval=yes ;GS
Armor=light
TechLevel=-1
Sight=7
Speed=6
Owner=British,French,Germans,Americans,Alliance
AllowedToStartInMultiplayer=no
Cost=1000
Turret=no ; can't have a turrett and a NoSpawnAlt (both go in AuxVoxel)
Points=30
ROT=5
Crusher=yes
Weight=3
Crewed=no
Explosion=TWLT070,S_BANG48,S_BRNL58,S_CLSN58,S_TUMU60
VoiceSelect=
VoiceMove=
VoiceAttack=
VoiceFeedback=
DieSound=
SinkingSound=GenLargeWaterDie
Locomotor={2BEA74E1-7CCA-11d3-BE14-00104B62A16C}
SpeedType=Float
MovementZone=Water
ThreatPosed=15	; This value MUST be 0 for all building addons
DamageParticleSystems=SparkSys,SmallGreySSys
VeteranAbilities=STRONGER,FIREPOWER,ROF,SIGHT,FASTER
EliteAbilities=SELF_HEAL,STRONGER,FIREPOWER,ROF
TooBigToFitUnderBridge=true
Sensors=yes
SensorsSight=8
OpportunityFire=yes ; since no turret, will only apply to helicopter (since ASWLauncher has OmniFire)
ElitePrimary=155mmE
```

This was merged without prerequisites.
Please stop merging things without checking for completeness.

**Comment by Mailaender** [comment]
They aren't buildable, but neutral actors for decoration so they don't need it. Please review pull requests while they are open instead of moaning afterwards.

**Comment by GraionDilach** [comment]
The Coast Guard Boat never had any prereqs. i fail to see your point here.

**Comment by Phrohdoh** [comment]
The `Owner` tag should have been made into prereqs.
@Mailaender Maybe you should say something along the lines of "yeah looks good to me, going to merge if no one else has a problem with it."

**Comment by GraionDilach** [comment]
The `Owner` tag was only used for AI in RA2 mostly (that, starting units and a way to trigger the NCO bug), I don't see either case justifying them converted on a one-on-one basis, since every such thing works differently in OpenRA in the levels `Owner` would affect.

Nevertheless, I'd label applying faction owners to civilian assets as a bug. Even if the said civilian asset is a lazy copy-paste of the Allied Destroyer and some unneeded tags were remained.

**Pull request merged**